src/Splat.Autofac/SplatAutofacExtension.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) 2019 .NET Foundation and Contributors. All rights reserved.
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using Autofac;
+
+namespace Splat.Autofac
+{
+    /// <summary>
+    /// Extension methods for the Autofac adapter.
+    /// </summary>
+    public static class SplatAutofacExtension
+    {
+        /// <summary>
+        /// Initializes an instance of <see cref="AutofacDependencyResolver"/> to override the default <see cref="Locator"/>.
+        /// </summary>
+        /// <param name="container">Autofac container.</param>
+        /// <returns>The dependency resolver.</returns>
+        public static AutofacDependencyResolver UseAutofacDependencyResolver(this IContainer container)
+        {
+            var resolver = new AutofacDependencyResolver(container);
+            resolver.InitializeSplat();
+            return resolver;
+        }
+
+        /// <summary>
+        /// Initializes an instance of <see cref="AutofacDependencyResolver"/> to override the default <see cref="Locator"/>.
+        /// </summary>
+        /// <param name="containerBuilder">Autofac container builder.</param>
+        /// <returns>The dependency resolver.</returns>
+        public static AutofacDependencyResolver UseAutofacDependencyResolver(this ContainerBuilder containerBuilder)
+        {
+            var container = containerBuilder.Build();
+            var resolver = new AutofacDependencyResolver(container);
+            resolver.InitializeSplat();
```

Comment:
We may have to remove the auto-register thingy in ReactiveUI if we do this to avoid duplicates.

Reply:
In my implementation prior to the fix in splat, I was calling
 ```
resolver.InitializeSplat();
resolver.InitializeReactiveUI();
```

So we are no longer expecting our consumers to call either?  Then there is very little need for the wrapper extension.